lib/services/downloads_service.dart

```diff
@@ -525,9 +534,10 @@ class DownloadsService {
   }
 
   /// type needs to be a key of the itemDeletedSnackbar replacement
-  /// /// As of writing this code the following options exist: album, playlist, artist, genre, song, library
+  /// As of writing this code the following options exist: album, playlist, artist, genre, song, library
```

Comment:
this could be converted to an enum (or maybe there already is a type for that!)

Reply:
I also changed this to not be a required parameter anymore, dont really know why this should have been a parameter in the first place (will push later)